src/ducks/connections/test/connections.spec.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-env jest */
+
+import connections, {
+   createConnection,
+   updateConnectionRunningStatus
+ } from '../'
+
+describe('Connections Duck', () => {
+  describe('createConnection', () => {
+    it('adds new connection in empty state', () => {
+      const state = undefined
+      const konnector = { slug: 'cozy' }
+      const account = { _id: '9bf93550308311c59f0a0047fc00fa1b' }
+
+      const result = connections(state, createConnection(konnector, account))
+
+      expect(result).toEqual({
+        cozy: {
+          '9bf93550308311c59f0a0047fc00fa1b': {}
+        }
+      })
```

Comment:
You can use Jest snapshots here.

Reply:
Never heard of that. I am going to have a look.

Reply:
@enguerran Ok got it. I thank it was only for components testing.